elastic/kibana
# [Uptime] Handle filter dropdown button click

## Summary

Related to https://github.com/elastic/kibana/issues/48638.

When the user clicks the button that opened the filter dropdown to close it, the page does not update and the selected filters are not applied. This patch adds the handler to the click event in addition to the popover close.

## Testing this change
0. Start up the Uptime/Heartbeat solution
1. Apply filters from a dropdown
2. Click the link and make sure your filters are applied and the UI updates

### Checklist

Use ~~strikethroughs~~ to remove checklist items you don't feel are applicable to this PR.


### For maintainers
